MetadataExtractor/Formats/Riff/RiffReader.cs

```diff
@@ -70,7 +70,7 @@ private static void ProcessChunks(SequentialReader reader, long maxPosition, IRi
                 {
                     if (chunkSize < 4)
                         break;
-                    string listName = reader.GetString(4, Encoding.ASCII);
+                    ReadOnlySpan<byte> listName = reader.GetBytes(4);
```

Comment:
I avoided a stackalloc in this case, since we're in a while statement. but let me refactor this to eliminate this safely.

<img width="475" alt="Screenshot 2024-01-29 at 7 01 00 PM" src="https://github.com/drewnoakes/metadata-extractor-dotnet/assets/7537/30ac6b3f-4580-43da-96a6-4eb0f4b242fc">

Reply:
Updated!